machine-learning/ml_tips/queries/train_randomforest_classifier.sql

```diff
@@ -0,0 +1,10 @@
+-- client: molehill/0.0.1
+select
+  train_randomforest_classifier(
+    features
+    , survived
+    , '-trees 15 -seed 31 -attrs Q,Q,C,C,C'
```

Comment:
Once I've tried to use sparse input, but it is designed for real "sparse vector," which means it doesn't perform accurate as dense input since the algorithm for sparse vector is different from for dense vector. So I decide to use dense input.

Reply:
Add note 997d035